src/RateTimeInTransit.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+namespace Ups;
+
+use DOMDocument;
+use DOMElement;
+use Exception;
+use SimpleXMLElement;
+use Ups\Entity\RateRequest;
+use Ups\Entity\RateResponse;
+use Ups\Entity\Shipment;
+
+/**
+ * RateTimeInTransit API Wrapper.
+ */
+class RateTimeInTransit extends Rate
+{
+    /**
+     * @param $rateRequest
+     *
+     * @throws Exception
+     *
+     * @return RateResponse
+     */
+    public function getRateTimeInTransit($rateRequest)
+    {
+        if ($rateRequest instanceof Shipment) {
```

Comment:
Can we skip this part and just require `getRateTimeInTransit` to only accept 'RateRequest' object?

Reply:
I like the idea of specifying the type as a parameter. I think @mazzarito was just copying what's happening in Rate though. Should we keep RateTimeInTransit similar?

Reply:
I see, I can live with that.